### DISCUSSION TITLE
[FEATURE REQ] Auto-delete releases older than X hours/days/months

### DISCUSSION THREAD

**Discussion by onedr0p** [autobrr/autobrr#1686] [Ideas]
**Is your feature request related to a problem? Please describe.**

Follow up to https://github.com/autobrr/autobrr/pull/924

It would be great if releases could automatically be purged after X amount of time has passed.

For my use-case I would love it if I could automatically purge releases older than a week or month.

**Describe the solution you'd like**

Allow choosing a configurable amount of time to pass before releases older than X hours/days/months are automatically pruned.

**Describe alternatives you've considered**

Scripting this with a sidecar container that can access the database and run SQL commands. However this isn't ideal and could break on future updates to autobrr. Having it directly integrated as a feature would be awesome!

**Additional context**

❤️

**Comment by zze0s** [comment]
There is actually an API endpoint for it, so no need to touch SQL ;)

```sh
curl 'http://autobrr/api/release?olderThan=168?apikey=MYAPIKEY' -X DELETE
```

The value for `olderThan` is hours so it's pretty flexible.

Having it as a scheduled job was the plan from the start but we started with manual.

**Comment by gravelfreeman** [comment]
Same issue here. Logs are taking too much space over time. With volsync cache it becomes a lot.

**Comment by mrgeetv** [comment]
This functionality has been released. See https://github.com/autobrr/autobrr/releases/tag/v1.72.0 and https://github.com/autobrr/autobrr/pull/2247.
